### PR TITLE
Add Web3 to the glossary

### PR DESCRIPTION
**Pull request by MrBrain295**
#### Summary
This adds Web3 to the glossary.

#### Motivation
I saw the glossary didn’t have Web3.

#### Supporting details
None.

#### Related issues
None.

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/glossary/web3/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Web3
+slug: Glossary/web3
+tags:
+  - Blockchain
+  - Glossary
+  - Infrastructue
+---
+Web3 is a web controled by the people through a blockchain; making them nearly imossibe to be controlled by a governmet or comany. They also have a currency for the blockchain they use.
```
Comment (Yatin-kathuria) [inline files/en-us/glossary/web3/index.md:9]:
```Suggestion
Web3 is a web controlled by the people through a blockchain; making them nearly impossible to be controlled by a government or company. They also have a currency for the blockchain they use.
```

File: files/en-us/glossary/web3/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+For example: 
+- [Uniswap](https://uniswap.org) a web3 appthat allows you to swap one cryptocurrency for another and provide liquidity for cryptocurrency.
```
Comment (Yatin-kathuria) [inline files/en-us/glossary/web3/index.md:12]:
```Suggestion
- [Uniswap](https://uniswap.org) is a web3 app that allows you to swap one cryptocurrency for another and provide liquidity for cryptocurrency.
```

File: files/en-us/glossary/web3/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- [Curve](https://curve.fi) a web3 smart contract that allows users to provide liqudity to different cryptocurrencys.
```
Comment (Yatin-kathuria) [inline files/en-us/glossary/web3/index.md:13]:
```suggestion
- [Curve](https://curve.fi) is a web3 smart contract that allows users to provide liquidity to different cryptocurrencies.
```

**Comment by wbamberg** [comment]
Thanks for the PR, but at this point we feel Web3 isn't clearly enough defined for a sensible glossary entry on MDN.

**Pull request closed without merging**